txp/Pos/Txp/Toil/Failure.hs

```diff
@@ -89,8 +91,10 @@ instance Buildable ToilVerFailure where
         tInputSum tOutputSum
     build (ToilInconsistentTxAux msg) =
         bprint ("TxAux is inconsistent: "%stext) msg
-    build (ToilInvalidOutputs msg) =
-        bprint ("outputs are invalid: "%stext) msg
+    build (ToilInvalidOutputs i reason) =
+        bprint ("input #"%int%"'s outputs are invalid:\n'"%
```

Comment:
I would say `ords%" output is invalid:\n"%`, see [ords](https://hackage.haskell.org/package/formatting-6.3.1/docs/Formatting-Formatters.html#v:ords). It's not related to _input_.